src/Components/Promises/PromiseTableAdminPanel/index.js

```diff
@@ -0,0 +1,80 @@
+/* @flow */
+import React from 'react';
+import SubjectSelector from '../../Subjects/SubjectSelector';
+import RaisedButton from 'material-ui/RaisedButton';
+import type { SubjectId } from '../../../types';
+import { createSubject } from '../../../utils/api';
+
+const styles = {
+  container: {
+    display: 'flex',
+  },
+  select: {
+    flexGrow: 1,
+  },
+  buttonContainer: {},
+};
+
+type Props = {
+  linkSubjectsCallback: (subjectId: SubjectId) => void,
+};
+
+type State = {
+  subject: ?string | SubjectId,
+};
+
+type ReactSelectValue = {
+  value: ?string | number,
+};
+
+export class PromiseTableAdminPanel extends React.Component<void, Props, State> {
+  props: Props;
+  state = {
+    subject: null,
+  };
+
+  updateSubject = (selectedValue: ?ReactSelectValue) => {
+    const subject = selectedValue && selectedValue.value;
```

Comment:
add a default value here with || ? So it doesnt default to false

Reply:
It'll be `null` if `selectedValue` is not set.

Try this in your browser 
```
const selectedValue = null;
const subject = selectedValue && selectedValue.value
console.log(subject)
```

Reply:
I was more thinking if selectedValue is truthy and selectedValue.value is falsy, wont it then become false?

Reply:
seems to be undefined if value is not a defined key in selectedValue

Reply:
Correct, if value is `undefined` then the `subject` will be undefined as well.

But this will "reset" the value in `setState`

Basically it wont end up as a boolean